package core;
import java.util.InputMismatchException;
import java.util.Scanner;

class Divide{
	public void division() {
		Scanner sc = new Scanner(System.in);
		System.out.println("enter a = ");
		int a = sc.nextInt();
		System.out.println("enter b = ");
		int b = sc.nextInt();
		int c = a / b;
		System.out.println("c = " + c);
	}
}

public class P018_Exception {
	public static void main(String[] args) {
		Divide d = new Divide();
		d.division();
		
		
		
		
		
		
		
//		try {
//			Scanner sc = new Scanner(System.in);
//			System.out.println("enter a = ");
//			int a = sc.nextInt();
//			System.out.println("enter b = ");
//			int b = sc.nextInt();
//			int c = a / b;
//			System.out.println("c = " + c);
//			System.exit(0);
//		} catch (ArithmeticException e) {
//			System.out.println("denominator should be > 0");
//		} catch (InputMismatchException e) {
//			System.out.println("dnominator should be numeric value");
//		} catch (Exception e) {
//			e.printStackTrace();
//		}
//		finally {
//			System.out.println("this will executes everytime");
//		}
	}
}
